crosswalks-sssom/zenodo/zenodo-codemeta-mappings.yml

```
@@ -0,0 +1,14 @@
+---
+curie_map: # This contains the URLs prefixes used in the mapping file
+  skos: http://www.w3.org/2004/02/skos/core# ### Always needed. Do not modify
+  sssom: https://w3id.org/sssom/             ### Always needed. Do not modify
+  subject: "https://github.com/zenodo/zenodo/blob/master/zenodo/modules/deposit/jsonschemas/deposits/records/legacyrecord.json"
+  subject_documentation: "https://developers.zenodo.org/"
+  schema: "http://schema.org/" ### Always needed. Do not modify
+  codemeta: "https://codemeta.github.io/terms/" ### Always needed. Do not modify
+license: https://creativecommons.org/publicdomain/zero/1.0/
+mapping_date: "2025-04-10"
+mapping_set_description: "Zenodo metadata mapping to Codemeta"
+mapping_provider: "0000-0002-5082-6404"
```

```suggestion
mapping_provider: "https://orcid.org/0000-0002-5082-6404"
```
just for consistency (we are expecting URLs)